README-DEV.rst

```diff
@@ -115,15 +115,18 @@ Unless you want to install the optional dependencies, skip to the next section.
 
 Install what else we can with Homebrew::
 
-    brew install help2man gnupg md5sha1sum node python
+    brew install \
+      erlang \
+      elixir \
+      help2man \
+      node \
+      python \
+      spidermonkey
 
-If you don't already have pip installed, install it::
-
-    sudo easy_install pip
 
 Now, install the required Python packages::
 
-    sudo pip install sphinx docutils pygments sphinx_rtd_theme
+    pip3 install sphinx docutils pygments sphinx_rtd_theme
 
```

Comment:
Technically for doc for local dev just having python3 with pip is enough. We do most of the requirements in a venv anyway https://github.com/apache/couchdb/tree/main/src/docs https://github.com/apache/couchdb/blob/main/src/docs/requirements.txt

Reply:
or are you suggesting that whole line can go? (suggested inline patches = <3 ;)

Reply:
I am not sure even if Python needs to be installed via `brew`.  The `xcode-select --install` command would add Python 3 and `pip3`, which are of versions 3.9.6 and 21.2.4 respectively at the moment (Ventura 13.1).

Reply:
I ran into issues with the built-in python

Reply:
I think we can update the [src/docs](https://github.com/apache/couchdb/tree/main/src/docs) to use `pip3`.  I probably just had pip linked to it.

Reply:
what do y’all want me to do with this line?

Reply:
Let's merge it. It looks good. I have PR for docs setup+readme to use pip3 instead of pip https://github.com/apache/couchdb/pull/4415